unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/FavoritesSubSection/FavoritesesSubSectionComponentController.cs

```diff
@@ -0,0 +1,174 @@
+    }
+
+    private void OnRequestedEventsUpdated(List<HotSceneInfo> placeList)
+    {
+        friendsTrackerController.RemoveAllHandlers();
+
+        favoritesFromAPI = placeList;
+
+        view.SetFavorites(PlacesAndEventsCardsFactory.CreatePlacesCards((TakeAllForAvailableSlots(placeList))));
+        view.SetShowMoreFavoritesButtonActive(availableUISlots < favoritesFromAPI.Count);
+    }
+
+    internal List<HotSceneInfo> TakeAllForAvailableSlots(List<HotSceneInfo> modelsFromAPI) =>
+        modelsFromAPI.Take(availableUISlots).ToList();
+
+    internal void ShowMoreFavorites()
+    {
+        int numberOfExtraItemsToAdd = ((int)Mathf.Ceil((float)availableUISlots / view.currentFavoritePlacesPerRow) * view.currentFavoritePlacesPerRow) - availableUISlots;
+        int numberOfItemsToAdd = (view.currentFavoritePlacesPerRow * SHOW_MORE_ROWS_INCREMENT) + numberOfExtraItemsToAdd;
+
+        List<HotSceneInfo> placesFiltered = availableUISlots + numberOfItemsToAdd <= favoritesFromAPI.Count
+            ? favoritesFromAPI.GetRange(availableUISlots, numberOfItemsToAdd)
+            : favoritesFromAPI.GetRange(availableUISlots, favoritesFromAPI.Count - availableUISlots);
+
+        view.AddFavorites(PlacesAndEventsCardsFactory.CreatePlacesCards(placesFiltered));
+
+        availableUISlots += numberOfItemsToAdd;
+
+        if (availableUISlots > favoritesFromAPI.Count)
+            availableUISlots = favoritesFromAPI.Count;
+
+        view.SetShowMoreFavoritesButtonActive(availableUISlots < favoritesFromAPI.Count);
+    }
+
+    internal void ShowPlaceDetailedInfo(PlaceCardComponentModel placeModel)
+    {
+        view.ShowPlaceModal(placeModel);
+        exploreV2Analytics.SendClickOnPlaceInfo(placeModel.hotSceneInfo.id, placeModel.placeName);
+
+        dataStore.exploreV2.currentVisibleModal.Set(ExploreV2CurrentModal.Places);
+    }
+
+    internal void OnJumpInToPlace(HotSceneInfo placeFromAPI)
+    {
+        JumpInToPlace(placeFromAPI);
+        view.HidePlaceModal();
+
+        dataStore.exploreV2.currentVisibleModal.Set(ExploreV2CurrentModal.None);
+        OnCloseExploreV2?.Invoke();
+        exploreV2Analytics.SendPlaceTeleport(placeFromAPI.id, placeFromAPI.name, placeFromAPI.baseCoords);
+    }
+
+    private void View_OnFriendHandlerAdded(FriendsHandler friendsHandler) =>
+        friendsTrackerController.AddHandler(friendsHandler);
+
+    private void OnChannelToJoinChanged(string currentChannelId, string previousChannelId)
+    {
+        if (!string.IsNullOrEmpty(currentChannelId))
+            return;
+
+        view.HidePlaceModal();
+        dataStore.exploreV2.currentVisibleModal.Set(ExploreV2CurrentModal.None);
+        OnCloseExploreV2?.Invoke();
+    }
+
+    /// <summary>
+    /// Makes a jump in to the place defined by the given place data from API.
+    /// </summary>
+    /// <param name="placeFromAPI">Place data from API.</param>
+    public static void JumpInToPlace(HotSceneInfo placeFromAPI)
```

Comment:
I will remove this part for the moment because it's not yet defined how it will work in the places section, not even if we will have this info, as soon as it's defined I'll integrate it in a separate injected service